spec/helpers/alaveteli_pro/public_bodies_helper_spec.rb

```diff
@@ -0,0 +1,46 @@
+# -*- encoding : utf-8 -*-
+require 'spec_helper'
+
+describe AlaveteliPro::PublicBodiesHelper, type: :helper do
+  let(:public_body) { FactoryGirl.create(:public_body) }
+
+  describe '#public_body_search_attributes' do
+    let(:html) { double(:html) }
+    let(:expected) do
+      {
+        id: public_body.id,
+        name: public_body.name,
+        short_name: public_body.short_name,
+        notes: public_body.notes,
+        info_requests_visible_count: public_body.info_requests_visible_count,
+        html: html
+      }
+    end
+
+    before do
+      # Stubbing this so we can test both branches of this conditional -
+      # In reality from within an view, `render` calls seems to internally uses
+      # `render_to_string` but we just can't call it directly
+      allow(helper).to receive(:respond_to?).with(:render_to_string)
+        .and_return(in_controller)
```

Comment:
We use trailing dot for multi-line method chaining

Reply:
We should enabled Rubocop's `Layout/DotPosition` with `trailing` style then